### ISSUE TITLE
windows binary dist lack vcdll, and `hello world` not work

### ISSUE THREAD

**Issue by di3663** [ziglang/zig#2253]
please support win32 instead of win64 only

```
debug.zig:184:55: error: expected type 'builtin.AtomicRmwOp', found 'builtin.AtomicRmwOp'
```

**Comment by emekoi** [comment]
win32 was initially supported, but support dropped to tier 2 because of a lack of ci and other errors in the stdlib iirc.

**Comment by tgschultz** [comment]
See: #537

**Comment by andrewrk** [comment]
I would be happy to help you figure out your error, but I need more details. Feel free to provide more details and I will re-open this issue.

**Comment by xiannox** [comment]
I'm trying out zig 0.4.0 on 64-bit linux (artix, package details [here](https://www.archlinux.org/packages/community/x86_64/zig/)), and am getting the same error.

```
$ zig init-exe
Created build.zig
Created src/main.zig

Next, try `zig build --help` or `zig build run`
$ zig build
/usr/lib/zig/std/debug.zig:184:55: error: expected type 'builtin.AtomicRmwOp', found 'builtin.AtomicRmwOp'
    if (@atomicRmw(u8, &panicking, builtin.AtomicRmwOp.Xchg, 1, builtin.AtomicOrder.SeqCst) == 1) {
                                                      ^
/usr/lib/zig/std/special/panic.zig:22:33: note: called from here
            std.debug.panicExtra(error_return_trace, first_trace_addr, "{}", msg);
                                ^
/home/<username>//.local/share/zig/stage1/builtin/IEAsAbLEYTxXMpmtN5qXr4ThVenooCaLuzEFEPTVBUGABhhzuHS5n06-UiVGB8rh/builtin.zig:188:25: note: builtin.AtomicRmwOp declared here
pub const AtomicRmwOp = enum {
                        ^
/home/<username>/.local/share/zig/stage1/builtin/IEAsAbLEYTxXMpmtN5qXr4ThVenooCaLuzEFEPTVBUGABhhzuHS5n06-UiVGB8rh/builtin.zig:188:25: note: builtin.AtomicRmwOp declared here
pub const AtomicRmwOp = enum {
                        ^

```

I have tried deleting ~/.local/share/zig in case it contains something from previous versions, but error persists. Please advise.

**Comment by andrewrk** [comment]
It looks like a bug in the path canonicalization. `/home/<username>//` (extra slash) vs `/home/<username>/`. Fixed in 1d95fdaf6e952674514fe5692ebd3fe1834926cb.
